components/mosart/src/riverroute/MOSART_physics_mod.F90

```diff
@@ -180,6 +180,7 @@ subroutine Euler
 
        call t_startf('mosartr_subnetwork')    
        TRunoff%erlateral(:,:) = 0._r8
+	   if (heatflag) THeat%ha_lateral(:) = 0._r8
```

Comment:
Can you please indent the code and remove additional spaces at the beginning?

Reply:
Btw, why not initialize `THeat%ha_lateral(:)` irrespective of `heatflag`?

Reply:
Done. I also replaced the other tabs in this file with spaces. 

THeat%ha_lateral(:) has already been initialized as zeros when allocated memory in RtmMod.F90. Here, when heatflag is TRUE, THeat%ha_lateral(:) needs to be set to zeros again each time Euler() is called; when heatflag is FALSE, it will not be invoked so no need to set to zero anymore.